config/samples/storage_v1_csm_powerflex.yaml

```diff
@@ -129,6 +129,8 @@ spec:
 
         # X_CSI_MAX_VOLUMES_PER_NODE: Defines the maximum PowerFlex volumes that can be created per node
         # Allowed values: Any value greater than or equal to 0
+        # If value is zero CO SHALL decide how many volumes of this type can be published by the controller to the node.
```

Comment:
Please expand CO and "SHALL" need not be in capitals. Same comment for the below file

Reply:
Done